packages/camera/camera_android_camerax/android/build.gradle

```diff
@@ -22,14 +22,38 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 32
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
     defaultConfig {
-        minSdkVersion 16
+        // Many of the CameraX APIs require API 21.
+        minSdkVersion 21
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
+dependencies {
+    // CameraX core library using the camera2 implementation must use same version number.
+    def camerax_version = "1.2.0-alpha04"
```

Comment:
This cirrus warning are complaining that this is not version `1.2.0-beta01`. Is there a reason you chose this version?

Reply:
That was the latest at the time I wrote it. I'll change it to the latest!